registry.k8s.io/manifests/k8s-staging-cluster-api-vsphere/promoter-manifest.yaml

```diff
@@ -0,0 +1,44 @@
+# google group for gcr.io/k8s-staging-cluster-api-vsphere is k8s-infra-staging-cluster-api-vsphere@kubernetes.io
+registries:
```

Comment:
@ameukam Pretty sure the changes in the other files are correct

Do I need this change?

Reply:
Just found the doc. looks like it is correct